.github/workflows/compile.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run detekt
+      run: ./gradlew detektCheck
```

Comment:
I think it's wrong

Reply:
will revert the commit so it goes back to the first config, didn't think that the other pull was needed

Reply:
Yes, it was my bad. I had to tell you that in the description but I forgot it!